### PR TITLE
Twig ComputerVirtualMachine

### PR DESCRIPTION
**Pull request by stonebuzz**
Use ```twig``` template for ```ComputerVirtualMachine```

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

File: src/ComputerVirtualMachine.php [hunk 1]
```diff
@@ -115,7 +164,7 @@ public function post_getEmpty()
      *
      * @return true if displayed  false if item not found or not right to display
      **/
-    public function showForm($ID, array $options = [])
+    public function showFormv1($ID, array $options = [])
```
Comment (trasher) [inline src/ComputerVirtualMachine.php:167]: Should not this one be removed?

Comment (stonebuzz) [reply src/ComputerVirtualMachine.php]: indeed =)

**Review by trasher — APPROVED** [review]

File: src/ComputerVirtualMachine.php [hunk 2]
```diff
@@ -106,18 +110,15 @@ public function post_getEmpty()
 
 
     /**
-     * Print the version form
+     * Display form for antivirus
```
Comment (cedric-anne) [inline src/ComputerVirtualMachine.php:113]:
```suggestion
     * Display form
```

File: src/ComputerVirtualMachine.php [hunk 3]
```diff
@@ -106,18 +110,15 @@ public function post_getEmpty()
...
      *
-     * @param $ID        integer ID of the item
-     * @param $options   array
-     *     - target for the Form
-     *     - computers_id ID of the computer for add process
+     * @param integer $ID      id of the antivirus
```
Comment (cedric-anne) [inline src/ComputerVirtualMachine.php:115]:
```suggestion
     * @param integer $ID
```

File: templates/components/form/computervirtualmachine.html.twig [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                     item.fields['comment'],
+                     _n('Comment', 'Comments', get_plural_number()),
+                     field_options
+                  ) }}
+
+                  {{ fields.dropdownField(
+                     'VirtualMachineType',
+                     'virtualmachinetypes_id',
+                     item.fields['virtualmachinetypes_id'],
+                     _n('Virtualization system', 'Virtualization systems', get_plural_number()),
```
Comment (cedric-anne) [inline templates/components/form/computervirtualmachine.html.twig:89]:
```suggestion
                     'VirtualMachineType'|itemtype_name,
```

File: templates/components/form/computervirtualmachine.html.twig [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+                     field_options
+                  ) }}
+
+                  {{ fields.dropdownField(
+                     'VirtualMachineSystem',
+                     'virtualmachinesystems_id',
+                     item.fields['virtualmachinesystems_id'],
+                     _n('Virtualization model', 'Virtualization models', get_plural_number()),
```
Comment (cedric-anne) [inline templates/components/form/computervirtualmachine.html.twig:97]:
```suggestion
                     'VirtualMachineSystem'|itemtype_name,
```

File: templates/components/form/computervirtualmachine.html.twig [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+                     field_options
+                  ) }}
+
+                  {{ fields.dropdownField(
+                     'VirtualMachineState',
+                     'virtualmachinestates_id',
+                     item.fields['virtualmachinestates_id'],
+                     _n('State of the virtual machine', 'States of the virtual machine', get_plural_number()),
```
Comment (cedric-anne) [inline templates/components/form/computervirtualmachine.html.twig:105]:
```suggestion
                     'VirtualMachineState'|itemtype_name,
```

**Review by cedric-anne — CHANGES_REQUESTED** [review]

File: templates/components/form/computervirtualmachine.html.twig [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+                     computer.getTypeName(1),
+                     field_options
+                  ) }}
+
+                  {% set is_dynamic_value = call('Dropdown::getYesNo', [item.fields['is_dynamic']]) %}
+                  {{ fields.field('is_dynamic', is_dynamic_value, __('Automatic Inventory')) }}
+
+                  {{ fields.textField(
+                     'name',
+                     item,
```
Comment (cedric-anne) [inline templates/components/form/computervirtualmachine.html.twig:72]:
```suggestion
                     item.fields['name'],
```

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**